### DISCUSSION TITLE
Not able to find the github copilot extension on Visual Studio 2022

### DISCUSSION THREAD

**Discussion by VERMANIK7263** [community/community#18566] [Copilot Conversations] [answered]
Hi,

I subscribed for copilot extension, got an email few days back for an invite to test the GitHub copilot preview. Unfortunately , I cannot find it on manage extensions tab when I search for it. Anything I am missing ???

![image](https://user-images.githubusercontent.com/104910290/173550099-4b26e9c2-e8ce-4fab-a3e7-04aec94ee94f.png)

**Comment by kelvinelove** [comment]
Try [following this link](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) in your browser. It should prompt you to open in Visual Studio / VSCode. 

It should load the extension installation page from within your IDE.

https://marketplace.visualstudio.com/items?itemName=GitHub.copilot

**Reply by VERMANIK7263** [reply]
Ok, it takes me to VS code , but I wanted to install on VS 2022 which I actively use.

**Reply by kelvinelove** [reply]
Okay.

Try [this link](https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs)

That's the page specific to visual studio, so hopefully it should open in VS22:

https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by VERMANIK7263** [reply]
Thanks. Still having issues. It seems it is only supported for VS Community and I am using VS professional.

![image](https://user-images.githubusercontent.com/104910290/173559454-d324229d-02b2-4fa6-b00d-373e9269229c.png)

![image](https://user-images.githubusercontent.com/104910290/173559552-b17c9812-6a54-447a-9898-0eaa5ad60775.png)

**Reply by kelvinelove** [reply]
I guess so. I hope they release a version that works with VS Professional soon.

**Reply by shaair** [reply]
try with another version of Visual Studio. For me also at Community version not working.

**Reply by zzkatz** [reply]
- Go refresh your project , its conflicting with store ! I had same issue , fixed with a simple refresh

**Comment by hsenag** [comment]
What version of VS2022 do you have? Copilot supports 17.1 and up. It should support all editions, not just Community.

**Reply by kelvinelove** [reply]
He has 17.1x, according to the screenshot above

**Comment by philipkaare** [comment, marked as answer]
I just had the same problem, upgrading to 17.2 solved the problem.

**Reply by Guilderm** [reply]
same here, upgrading solved my problem

**Reply by djmeacha** [reply]
Thank you, I had the same problem.

This would be useful information to include on the page the tells you how to install it.

**Comment by hsenag** [comment]
Hi all, we've decided to drop support for Visual Studio 17.1 as it's not a long-term supported release and some problems seem to be emerging with it. If you are having problems with 17.2 or above, please could you start a new discussion?

**Reply by shaair** [reply]
Yes, it's working now. I updated visual studio from 17.0.2 to 17.2.5.

**Comment by kacperks** [comment]
i have this same problem but in vscode and im using arch linux

**Reply by devnulls** [reply]
I had to combine these two for vscode oss running on arch:
https://hakanu.net/python/2021/11/04/problems-with-github-copilot-and-visual-studio-code-oss-release/
and https://forum.manjaro.org/t/no-extension-named-copilot-on-vscode-installed-from-community-repository/103493
(yay -S code-marketplace)
